docs/BigQueryMultiTable-batchsink.md

```diff
@@ -39,6 +39,8 @@ bucket will be created and then deleted after the run finishes.
 **Truncate Table:** Whether or not to truncate the table before writing to it.
 Should only be used with the Insert operation.
 
+**Location:** The location where the big query datasets will get created. This value is ignored for existing dataset.
```

Comment:
nit: for existing dataset -> if the dataset already exists

same for other places

Reply:
actually, this is also ignored if the temporary bucket already exists.

if the dataset or temporary bucket already exist.